alvr/packets/src/lib.rs

```diff
@@ -59,14 +60,17 @@ impl VideoStreamingCapabilities {
     }
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct ConnectionAcceptedInfo {
+    pub client_protocol_id: u64,
+    pub platform: Platform,
```

Comment:
Also this kinda makes all the platform stuff part of the public api contract and so this needs to be either kept track of or there needs to be a comment that this isn't reliably info (or only partly) and should only be used for display purposes.

Reply:
Reverted to string